### PR TITLE
Fixing some typo

### PR DESCRIPTION
**Pull request by AnomalRoil**
This is fixing the typos I've found in #321.

### CODE REVIEW

**Comment by AnomalRoil** [comment]
@TelegramSam @awoie @tplooker : Not sure who's the lead editor, please review the above.

File: docs/spec-files/out_of_band.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ The `id` of the message passed in a URL or a QR code is used to as the `pthid` o
 
 Each message passed this way must be contained within an `out-of-band` message, as described below.
 
-The out-of-band protocol a single message that is sent by the *sender*.
+The out-of-band protocol consists in a single message that is sent by the *sender*.
```
Comment (TelegramSam) [inline docs/spec-files/out_of_band.md:23]: `consists of` instead of `consists in`?

**Review by TelegramSam — APPROVED** [review]
I like this set of changes.

File: docs/spec-files/out_of_band.md (hunk 1 above)
Comment (AnomalRoil) [reply docs/spec-files/out_of_band.md:23]: It seemed to be really an intrinsic property to me. Since you merged already I guess this is fine?

**Pull request merged**